modules/identify-dockerfile-line-component-cve.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+// * operating/examine-images-for-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="identify-dockerfile-line-component-cve_{context}"]
+= Identifying the Dockerfile line in an image that adds a component with a CVE
+
+[role="_abstract"]
+You can identify the specific Dockerfile line in an image that adds a component that has been identified as having a CVE. To view the problematic line:
+
+.Procedure
+
+. Navigate to the {product-title-short} portal and click *Vulnerability Management* from the navigation menu.
+. Select an image from the *Top Riskiest Images* widget or click the *Images* button at the top of the Dashboard and select an image.
+. In the *Image* details view, next to *Dockerfile*, select the expand icon next to *Image Findings*. 
+. Locate the CVE you want to examine further. In the *Affected Components* column, click on the *<number> Components* link to view a list of components affected by the CVE. You can perform the following actions in this window:
+* Select the expand icon next to a specific component to view the Dockerfile line in the image where the component was added that contains the CVE. To address the CVE, you need to change this line in the Dockerfile; for example, you can upgrade the component. 
+* Click the name of the component to go to the *Component Summary* page and view more information about the component. 
```

Comment:
These steps are incorrect. You do not use the Dockerfile section at all for the new functionality. You find the **Affected Components** column in the CVE tables in any of the Observed CVEs, Deferred CVEs, or False Positive CVEs tabs under the _Image Findings_ section.

Note in the screenshots below, I never opened the Dockerfile section at all.

![Screen Shot 2022-09-15 at 1 46 48 PM](https://user-images.githubusercontent.com/715729/190474550-ac8c21f0-06ee-49aa-8d15-8f15089ec0a8.png)

![Screen Shot 2022-09-15 at 1 46 57 PM](https://user-images.githubusercontent.com/715729/190474572-ba1a3053-cbd4-45e6-b2e8-b6549cdba68f.png)

![Screen Shot 2022-09-15 at 1 47 01 PM](https://user-images.githubusercontent.com/715729/190474591-fffa9741-3d73-4a46-9e89-1d4df5c27bbb.png)

Reply:
Oh, I made the new topic from the other file, so Step 1 is a cut and paste error. Let me fix & repush. Will also add column description (Observed, Deferred, False Positive).